src/core/render/index.js

```diff
@@ -226,7 +226,7 @@ export function initRender(vm) {
 
   if (el) {
     if (config.repo) {
-      html += tpl.corner(config.repo)
+      html += tpl.corner(config.repo, config.externalLinkTarget)
```

Comment:
as this is specific for GitHub corner, so `externalLinkTarget` name might not be appropriate. Better give it some specific name related to  github corner.

Reply:
thx for ur suggestion, I see what ur mean.

In my opinion. I think it is just the settings/options  of  the `link target attribute` at the corner.
no matter what kind of link at the corner ( such as `github` , `gitlab`  maybe in the future with the corner icon customized...etc.)
it is not based on the link type, so I mean it is unnecessary to make it related to each link type.

BUT, if the corner is  just for `github` now and then, I agree to give a specific name likes `githubExternalLinkTarget ` to it.

AND, I will  update the doc soon after this.

Reply:
`ExternalLinkTarget` might be confusing to others as it may tell that all the links present in the website will be in a new tab.

What do you think about `cornerExternalLinkTarge`

Reply:
Agree ,  I will change it and update the doc soon.

Reply:
Another question.
I remember that  `ExternalLinkTarget` is one of the [configuration](https://github.com/docsifyjs/docsify/blob/develop/docs/configuration.md#externallinktarget) in the docs now.  it works on those links which are in the page content.
```
externalLinkTarget
type: String
default: _blank
Target to open external links. Default '_blank' (new window/tab)

window.$docsify = {
  externalLinkTarget: '_self' // default: '_blank'
};
```
If we has signed another name of the corner, the corner would not be according to `externalLinkTarget` config. it would be a separate options.
Maybe we can think the `externalLinkTarget` is the **GLOBAL CONFIG** of both page content links and corner.

What's your opinion?

Reply:
Current `externalLinkTarget` is for the links inside the markdown.
So I guess you can have the cornerExternalLinkTarget name or anything which is not conflicting with this one and makes some understanding just by reading the name 👍 

Also, do try using `externalLinkTarget` to `_self` and your config option to `_blank` and vice versa and checking if there is any conflict between these two.

these two should be independent. 
_Which I think its currently as of your PR but just check one more time for verification_

Reply:
Okay, My currently PR is make `corner link` according on the `externalLinkTarget` config too, cuz I just add the `target` attribute in the corner's code.

I will try to make a new  `cornerExternalLinkTarget` as an  independent config for the corner and make sure it will not be conflicting  with the current `externalLinkTarget` config.